boot/mbed/CMakeLists.txt

```diff
@@ -2,9 +2,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Mbed-MCUboot Port
-
-cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
```

Comment:
this should still be here specifying the minimum version needed

Reply:
Technically this line is only needed in CMake scripts that can be executed as a top-level project.  But it's not doing any harm so I will re-add it.